pkgs/development/libraries/science/math/magma/generic.nix

```diff
@@ -182,7 +190,7 @@ stdenv.mkDerivation {
     # because it has no files to install.
     + ''
       install -Dm755 ./testing/testing_* ./sparse/testing/testing_* -t "$test/bin/"
-      install -Dm755 ./lib/libtester.so ./lib/liblapacktest.so -t "$test/lib/"
+      install -Dm755 ./lib/lib*test*.* -t "$test/lib/"
```

Comment:
Can also one day become empty.

Reply:
True, or the naming scheme they use for the test libraries could change. The fix for this is the same as you had mentioned previously -- upstreaming a patch for their CMake setup to recognize test components, right?

Reply:
Iirc yes. It also might be that upstream considers these tests "private" and doesn't provide a way to install them on purpose

Reply:
Tracked in #296286